server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -299,89 +295,167 @@ export class AlfredResourcesFactory implements core.IResourcesFactory<AlfredReso
 			);
 		};
 
+		const tenantThrottlersMap = new Map<string, string>();
+		const tenantsGroup1PerTenant = config.get("shared:tenantsGroup1PerTenant") as string[];
+		tenantsGroup1PerTenant.forEach((t) => tenantThrottlersMap.set(t, "tenantsGroup1PerTenant"));
+
+		const throttlersMap = new Map<string, Map<string, core.IThrottler>>();
```

Comment:
Can we please move all this code to a separate method in a separate .ts file or utils file?

The util methods can be 
1. getTenantGroupMap() and,
2. getThrottlerMap().

Reply:
I created the methods for throttlersMap.
I didn't create the tenant group map method, since it is straightforward.

Reply:
I created util methods for throttlers but doesn't do it for the tenant group, since it is straightforward.